Kernel/Arch/aarch64/RPi/MiniUART.cpp

```diff
@@ -61,7 +62,7 @@ UNMAP_AFTER_INIT ErrorOr<NonnullLockRefPtr<MiniUART>> MiniUART::create()
 }
 
 UNMAP_AFTER_INIT MiniUART::MiniUART()
-    : CharacterDevice(4, 64)
+    : CharacterDevice(MajorAllocation::CharacterDeviceNumbers::s_allocated_serial_devices_number, 64)
```

Comment:
Should the minor number still be 64, now that serial device numbers no longer start at 64?